apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/nux-modal/index.tsx

```diff
@@ -21,7 +23,18 @@ const NuxModal: React.FC< Props > = ( {
 	imageSrc,
 	actionButtons,
 	onRequestClose,
+	onOpen,
 } ) => {
+	const prevIsOpen = useRef< boolean | null >( null );
```

Comment:
Could this be a boolean starting from `isOpen`? or is the modal only loaded in the open state?
```suggestion
	const prevIsOpen = useRef< boolean >( isOpen );
```

Reply:
If we set the initial value to `isOpen` and the value is true at the beginning, the `onOpen` function won't be triggered with the current approach. We need to handle this case, so I make the initial value to `null` and it would be simple to handle it.